core/pluginz/ezpz_pluginz/__cli__.py

```diff
@@ -0,0 +1,706 @@
+# type: ignore[B008]
+
+import os
+import time
+import shutil
+from typing import Any
+from pathlib import Path
+
+import typer
+
+from ezpz_pluginz import mount_plugins, unmount_plugins
+from ezpz_pluginz.logger import setup_logger
+from ezpz_pluginz.registry import (
+  REGISTRY_URL,
+  LOCAL_REGISTRY_DIR,
+  LOCAL_REGISTRY_FILE,
+  PluginRegistryAPI,
+  LocalPluginRegistry,
+  install_package,
+  check_ezpz_config,
+  find_plugin_in_path,
+  is_package_installed,
+  setup_local_registry,
+  create_default_ezpz_config,
+)
+from ezpz_pluginz.toml_schema import load_config
+
+app = typer.Typer(name="ezplugins", pretty_exceptions_show_locals=False, pretty_exceptions_short=True)
```

Comment:
lets name the cli `ezpz` less chars to type and easier to remember. also notice that there is a theme of using `z` instead of `s` ie ezpz_pluginz vs ezpz_plugins